src/sentry/static/sentry/app/views/releases/detail/withCommitRepos.tsx

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router';
+import * as Sentry from '@sentry/react';
+
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {Client} from 'app/api';
+import Button from 'app/components/button';
+import {Body, Main} from 'app/components/layouts/thirds';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {IconCommit, IconWarning} from 'app/icons';
+import {t} from 'app/locale';
+import {Organization, Repository} from 'app/types';
+import getDisplayName from 'app/utils/getDisplayName';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+import withRepositories from 'app/utils/withRepositories';
+import EmptyMessage from 'app/views/settings/components/emptyMessage';
+
+import {ReleaseContext} from '.';
+
+// These props are required when using this HoC
+type DependentProps = RouteComponentProps<{orgId: string; release: string}, {}>;
+
+type HoCsProps = {
+  api: Client;
+  organization: Organization;
+  repositories?: Repository[];
+  repositoriesLoading?: boolean;
+  repositoriesError?: Error;
+};
+
+type State = {
+  commitRepositories: Repository[];
+  isLoading: boolean;
+  activeCommitRepository?: Repository;
+};
+
+const withCommitRepos = <P extends DependentProps>(
```

Comment:
Could call this `withReleaseRepos`?

Reply:
The name was because we are fetching all commits related repositories, but I agree, `withReleaseRepos` is a better name :) I've already updated the code